src/Provider/TaxonUrlProvider.php

```diff
@@ -24,7 +24,7 @@ class TaxonUrlProvider extends AbstractUrlProvider
 
     protected string $code = self::PROVIDER_CODE;
 
-    protected string $icon = 'folder';
+    protected string $icon = 'world';
```

Comment:
Is the `folder` icon missing in new admin css? It would be best to keep the icon which was initially chosen by the  plugin creators.